Sources/Load.php

```diff
@@ -2717,9 +2717,15 @@ function loadCSSFile($fileName, $params = array(), $id = '')
 	$params['validate'] = isset($params['validate']) ? $params['validate'] : true;
 	$params['order_pos'] = isset($params['order_pos']) ? (int) $params['order_pos'] : 3000;
 
-	// If this is an external file, automatically set this to false.
-	if (!empty($params['external']))
-		$params['minimize'] = false;
+	// Is this theme always external?
+	if ($themeRef == 'theme' && strpos($settings['theme_url'], $boardurl) !== 0)
```

Comment:
Hm. It just occurred to me that this check (and the identical one below) will cause problems if the admin decides to move the Themes directory outside of the main forum directory. So another solution is needed...